test/browser/time-picker/time-picker-spec.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import TextField from 'TextField';
 import TimePicker from 'TimePicker';
-import DateTime from 'utils/dateTime';
+import {addHours, formatTime} from 'TimePicker/timeUtils';
```

Comment:
@mbrookes After talking with @nathanmarks, seems like they can, but don't worry about it. We can do it another time.